dev-support/test-patch.d/whitespace.sh

```diff
@@ -16,6 +16,24 @@
 
 add_plugin whitespace
 
+
+function whitespace_linecomment_reporter
+{
+  local file=$1
+  local comment=$@
+  local tmpfile="${PATCH_DIR}/wlr.$$.${RANDOM}"
+
+  while read -r line; do
+    {
+      printf "${line}" | cut -f1-2 -d:
```

Comment:
shellcheck: 14: warning: Don't use variables in the printf format string. Use printf "..%s.." "$foo". [SC2059]